packages/ui-popover/src/Popover/README.md

```diff
@@ -9,25 +9,72 @@ Popovers hide or show content as a result of user interaction, such as clicking,
 ```js
 ---
 example: true
+render: false
 ---
-<Popover
-  renderTrigger={
-    <Link aria-describedby="tip">
-      Hover or focus me
-    </Link>
-  }
-  shouldRenderOffscreen
-  shouldReturnFocus={false}
-  placement="top center"
-  mountNode={() => document.getElementById('main')}
-  onPositioned={() => console.log('positioned')}
-  onShowContent={() => console.log('showing')}
-  onHideContent={() => console.log('hidden')}
->
-  <View padding="x-small" display="block" as="div" id="tip">
-    Hello World
-  </View>
-</Popover>
+class Example extends React.Component {
+    state = {
+        withArrow: true,
+        shouldAlignArrow: true,
+        color : "primary"
+    }
+    toggleWithArrow = (event) => this.setState({ withArrow: !this.state.withArrow })
+    toggleAlignArrow = (event) => this.setState({ shouldAlignArrow: !this.state.shouldAlignArrow })
+    changeColor = (event,color) =>  {this.setState({color})}
+    render() {
+        return (
+            <>
+                <FormFieldGroup description="Uncontrolled Popover Example">
```

Comment:
I would suggest making and extra example (called "Popover playground" or something like this) where the users can play with all the properties. The examples under the texts should focus on the the prop mentioned in the text only. 

(Also, as we talked about it, maybe you could move the inputs on a white panel to separate them from the example, using the View and Flex components)

Reply:
I just have done it, you should take a look.